### PR TITLE
More configurable map asset #17

### PR DESCRIPTION
**Pull request by petrabarus**
As mentioned in issue #17

### CODE REVIEW

File: MapAsset.php [hunk 1]
```diff
@@ -14,6 +14,24 @@
  *
  * Registers the google maps api
  *
+ * To update the key or other options like language, version, or library
```
Comment (tonydspaniard) [inline MapAsset.php]: Can you add this also on the `README.md` file?

**Comment by tonydspaniard** [comment]
Once you add this info to the README.md file, then I will merge. I should also add a CHANGELOG.md file

**Comment by petrabarus** [comment]
Updated README.md.

**Comment by tonydspaniard** [comment]
Great

**Pull request merged**